File: classes/parte-4-control-del-programa/064-iteracion-por-rango-for-clasico-y-for-range/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long n;
    if (scanf("%ld", &n) != 1) return 1;
    long f = 1;
    for (long i = 1; i <= n; i++) {
        f *= i;
    }
    printf("factorial=%ld\n", f);
    return 0;
}

File: classes/parte-4-control-del-programa/064-iteracion-por-rango-for-clasico-y-for-range/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	n, _ := strconv.Atoi(strings.TrimSpace(line))
	var f int64 = 1
	for i := 1; i <= n; i++ {
		f *= int64(i)
	}
	fmt.Printf("factorial=%d\n", f)
}
